cocos/ui/UIEditBox/iOS/CCUISingleLineTextField.mm

```diff
@@ -27,17 +27,16 @@ of this software and associated documentation files (the "Software"), to deal
 #import "CCUISingleLineTextField.h"
 #import "CCUITextInput.h"
 
-#include "base/CCDirector.h"
-
-
 @implementation CCUISingleLineTextField
 
 - (CGRect)textRectForBounds:(CGRect)bounds
 {
-    auto glview = cocos2d::Director::getInstance()->getOpenGLView();
+    if (!_leftPadding) {
```

Comment:
`!_leftPadding` is wired... I think compare float point should be something like `if(fabs(xxx) < FLT_EPSLON)`.